hash/msx1_cass.xml

```diff
@@ -16786,7 +17351,53 @@ license:CC0-1.0
 		</part>
 	</software>
 
+	<software name="wldmscpp">
+		<description>World Map Software for Color Plotter Printer (Japan)</description>
+		<year>1984</year>
+		<publisher>National</publisher>
+		<info name="alt_title" value="カラープロッタプリンタ用世界地図ソフト"/>
```

Comment:
Please don’t translate titles for descriptions – just use “Color Plotter/Printer-you yō Sekai Chizu Soft”.